codespell_lib/data/dictionary.txt

```diff
@@ -3722,26 +3937,64 @@ andles->handles, candles, angles, ankles, dandles,
 andling->handling, dandling,
 andoid->android
 andoids->androids
+andoird->android
+andoirds->androids
 andorid->android
 andorids->androids
 andriod->android
 andriods->androids
 androgenous->androgynous
 androgeny->androgyny
+androiders->androids
+androides->androids
 androidextra->androidextras
+androidos->androids
+androidtvs->androids
```

Comment:
Does this typo actually exist out there? 😀

Reply:
I'm constantly surprised by what I find out there, so I'd tend to trust `typos` here. 😄